postgres-full-text-search-engine.mdx

```diff
@@ -12,6 +12,8 @@ slug: postgres-full-text-search-engine
 
 This is part 1 of a blog mini-series, in which we explore the full-text search functionality in PostgreSQL and investigate how much of the typical search engine functionality we can replicate. In [part 2](https://xata.io/blog/postgres-full-text-search-postgres-vs-elasticsearch), we'll do a comparison between PostgreSQL's full-text search and Elasticsearch.
 
+> Hey there, you are on the Xata engineering blog. Xata is a serverless data platform that makes building on top of PostgreSQL and Elasticsearch really easy. [Check it out](https://xata.io/)!
```

Comment:
I changed to "Sign up today", but left it pointing to the home page.

I thought it works as a separate paragraph because the first one is talking about what this blog post is about. On other blog posts, I set it as the first one.